docs/json/sonarr/cf/anime-dual-audio.json

```diff
@@ -1,5 +1,6 @@
 {
   "trash_id": "418f50b10f1907201b6cfdf881f467b7",
+  "trash_score": "0",
```

Comment:
To my knowledge if a CF shouldn't get a score (0) then leave the score empty, Would need to ask @austinwbest if that's still the case,

Reply:
Spoke with Nit, you are correct have to remove the 0 scores